packages/turf-rhumb-bearing/index.js

```diff
@@ -0,0 +1,60 @@
+// https://en.wikipedia.org/wiki/Rhumb_line
+// http://www.movable-type.co.uk/scripts/latlong.html#rhumblines
+var getCoords = require('@turf/invariant').getCoords;
```

Comment:
Why then not renaming them `getCoords` and `getCoordsArray` (I guess this would be suitable for `LineString`, `Polygon`, etc. right?); I believe that would make it more clear and intuitive.